WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -109,6 +109,9 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
     // MARK: - Sorting
 
     private func setupSortingButton() {
+        guard FeatureFlag.readerSortingOption.enabled else {
```

Comment:
If `readerSortingOption` feature is not enabled I do not setup the sorting button.